app/errors.py

```diff
@@ -0,0 +1,19 @@
+class VaccineError(Exception):
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
+
+
+class NotWearingMaskError(Exception):
+    def __init__(self, friend_with_no_mask: int = 0) -> None:
+        self.friend_with_no_mask = friend_with_no_mask
+
+    def __str__(self) -> str:
+        return f"Friends should buy {self.friend_with_no_mask} masks"
```

Comment:
There is no need to implement this logic, place the required logic in the `go_to_cafe` and replace the code with `pass`.